config/checkstyle_non_main_files_checks.xml

```diff
@@ -25,8 +25,8 @@
       value="CI folder should only contain executables files"/>
   </module>
   <module name="RegexpOnFilename">
-    <property name="id" value="regexpOnGithubWorkflows"/>
-    <property name="folderPattern" value="[\\/].github[\\/]workflows([\\/]|$)"/>
+    <property name="id" value="regexpOnScriptsAndConfigs"/>
```

Comment:
I am not sure if it is better to have 1 module cover everything, but if we keep this, I would recommend something generic like `kebabCaseFileNaming`.

Reply:
At first, I went with another check, but they would be identical apart from the `folderPattern`.

I will rename to `kebabCaseFileNaming`.

Reply:
Done.

Reply:
> they would be identical apart from the folderPattern

Yes, but eventually the list may be become too big to maintain. Look at our naming checks in our config. Most of them use the exact same values, but keeping them separate makes it easier to see each identification and add more eventually.

Reply:
We are adding `config` folder after this PR. Not sure if more are planned to be added after that.

Alright, I will split.

Reply:
Done.